fern/apis/fern-definition/definition/auth.yml

```diff
@@ -86,6 +86,7 @@ types:
       client-id-env: optional<string>
       client-secret-env: optional<string>
       token-prefix: optional<string>
+      token-header: optional<string>
```

Comment:
nit: do you mind adding a `docs` here, once we fix json schema generation to include doc strings that will make the autocomplete nicer for users as well